src/views/Documentation/PageMeta.jsx

```diff
@@ -33,6 +33,41 @@ import { docsPageInfo } from '../../utils/prop-types';
   pageNavigationWithoutPrevious: {
     justifyContent: 'flex-end',
   },
+  smallText: {
+    fontSize: '10px',
+    color: 'gray',
+  },
+  largeText: {
+    color: '#000',
+    fontSize: '14px',
+  },
+  rowFlex: {
+    display: 'flex',
+  },
+  rightIcon: {
+    marginLeft: '7px',
+    marginRight: '0px',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+  rightButtonText: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    textAlign: 'left',
+  },
+  leftIcon: {
+    marginLeft: '0px',
+    marginRight: '7px',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+  },
+  leftButtonText: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    textAlign: 'right',
+  },
```

Comment:
`PageMeta` should not care about how to position anything that is inside the button. All of these styles should live in `PageTransition`.